packages/storybook/stories/wc-helpers.jsx

```diff
@@ -189,14 +189,16 @@ function NativeHandlers({ docsTags = [] }) {
 
   if (!handlers.length) return null;
 
+  const nativeEvents = handlers.join(', ');
+
   return (
     <div className="vads-u-margin-top--2">
-      This component uses the following native handlers:
-      <ul>
-        {handlers.map((handlerName, index) => (
-          <li key={index}>{handlerName}</li>
-        ))}
-      </ul>
+      This component uses the following native handlers:{' '}
+      <strong>{nativeEvents}</strong>. Please see our documentation on{' '}
+      <a href="https://design.va.gov/about/developers#native-events">
+        how to use web component native events
+      </a>
+      .
```

Comment:
I updated this to look like the custom events section for consistency. I can revert these changes if we don't want that.